Index: ncurses-6.4/configure
===================================================================
--- ncurses-6.4.orig/configure
+++ ncurses-6.4/configure
@@ -3735,7 +3735,7 @@ _ACEOF
 program_transform_name=`echo $program_transform_name | sed -f conftest.sed`
 rm conftest.sed
 
-for ac_prog in mawk gawk nawk awk
+for ac_prog in  gawk nawk awk
 do
   # Extract the first word of "$ac_prog", so it can be a program name with args.
 set dummy $ac_prog; ac_word=$2
